core/Dockerfile

```diff
@@ -58,4 +58,10 @@ FROM runner AS tester
 COPY jettyRunExtraFiles/mapfish-spring-application-context-override-acceptencetests.xml \
   webapps/ROOT/WEB-INF/classes/mapfish-spring-application-context-override.xml
 
+FROM runner AS watcher
+
+RUN apt update && \
+    apt install --yes python3-pip rsync
```

Comment:
missing cleanup of apt mess

Reply:
For me, we don't need to do that on an image that used only locally.
If you don't agree, no problem to add that...

Reply:
it still uses more place on the HD. Yesterday I cleaned 56Gb with a `docker system prune`

Reply:
Done :-)